<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_keywords">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>256</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>891</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_except">
     <attribute name="title">
      <string>제외 대상</string>
     </attribute>
     <widget class="QTableView" name="tv_exceptlist">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>871</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_scanadd">
      <property name="geometry">
       <rect>
        <x>805</x>
        <y>385</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>제외대상에서 삭제합니다.</string>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>검색포함</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_result">
     <attribute name="title">
      <string>탐색 결과</string>
     </attribute>
     <widget class="QTableView" name="tv_result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_scan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_except">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>제외대상 리스트에 추가합니다.</string>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>제외처리</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_move">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Download/Backup 폴더로 보관합니다.</string>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>보관</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delete">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>파일을 삭제합니다.</string>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>추가</string>
    </property>
   </widget>
   <widget class="QTreeView" name="tv_folder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>검색 파일 유형</string>
    </property>
    <widget class="QCheckBox" name="chk_ppt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>파워포인트(ppt,pptx)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_xls">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>엑셀(xls,xlsx)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_doc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>워드(doc,docx)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_pst">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>메일(pst)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_etc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>기타</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_vpx">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>vpx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_addext">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>201</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>대상 폴더</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_keyword">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>221</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>200</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_keyword">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>검색 대상 키워드(파일명)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
